src/DataProtection/DataProtection/src/KeyManagement/XmlKeyManager.cs

```diff
@@ -29,6 +30,7 @@ namespace Microsoft.AspNetCore.DataProtection.KeyManagement
     /// <summary>
     /// A key manager backed by an <see cref="IXmlRepository"/>.
     /// </summary>
+    [SupportedOSPlatform("windows")]
```

Comment:
Why is this one windows specific, it seems abstracted? Is it only used by windows specific storage systems?